pkg/master/thirdparty/thirdparty.go

```diff
@@ -207,6 +207,9 @@ func (m *ThirdPartyResourceServer) getExistingThirdPartyResources(path string) [
 				Name:       key,
 				Namespaced: true,
 				Kind:       obj.Kind(),
+				Verbs: metav1.Verbs([]string{
+					"delete", "deletecollection", "get", "list", "patch", "create", "update", "watch",
```

Comment:
while I think it is fine to hardcode this list, but how about checking the `entry.Storage` and see whether it supports all the verbs actually?